packages/bbui/src/Table/AttachmentRenderer.svelte

```diff
@@ -20,6 +20,9 @@
       target="_blank"
       download={attachment.name}
       href={attachment.url}
+      on:click={e => {
```

Comment:
I think `on:click|stopPropagation` may be equivalent here in svelte!

Reply:
Yep, I thought the same but the shorthand is reserved for native elements.